src/components/oscal-utils/OSCALRestUtils.js

```diff
@@ -238,3 +238,34 @@ export function createOrUpdateSspControlImplementationImplementedRequirementStat
     onError
   );
 }
+
+/**
+ * Creates and performs the REST request for adding a new SSP control implementation
+ * implemented requirement from the data.
+ *
+ * @param {*} partialRootRestData the partial root SSP object
+ * @param {*} newImplementedRequirement the new implemented requirement to add
+ * @param {*} onPreRestRequest function called just before making the REST request
+ * @param {*} onSuccess function called on a successful REST request with the result of the request as an argument
+ * @param {*} onError function called on error with the error as an argument
+ */
+export function createNewSspControlImplementation(
```

Comment:
We're technically creating a new implemented requirement, so to follow precedence above would recommend a more specific name of `createSspControlImplementationImplementedRequirement`.